components/avatar/avatar.css

```diff
@@ -0,0 +1,62 @@
+.avatar-basic__demo {
+  align-items: center;
+  gap: 1rem;
+  padding: 1rem;
+}
+.avatar {
+  width: fit-content;
+  height: fit-content;
+  position: relative;
+}
+.avatar__img {
+  width: 5.7rem;
+  height: 5.7rem;
+  border-radius: 50%;
+}
+.avatar__txt {
+  font-family: var(--primary-font-family);
+  font-size: 3rem;
+  padding: 1rem 2rem;
+  text-align: center;
+  vertical-align: middle;
+  background-color: var(--primary-color);
+  color: var(--white-color);
+  border-radius: 50%;
+}
+.material-icons.avatar__icon {
+  font-size: 40px;
```

Comment:
Use relative unit.

Reply:
hey ! its an icon used from the material icons stylesheet so thought using a absolute unit for it was the better way to work eith

Reply:
made the changes